tests/test_home.py

```
@@ -20,20 +23,10 @@
 
 
 @pytest.mark.asyncio
-@respx.mock
-async def test_home_from_api_with_7_6_175():
+async def test_home_from_api_with_7_6_175(mock_aioresponse: aioresponses) -> None:
```

Pylint (3.3.7)

[warning] Unused argument 'mock_aioresponse'

(W0613)